source/user_guide.rst

```diff
@@ -13,148 +14,465 @@ Who should read this
+            "id": 1
+          }
+        ],
+        "label": "Thermometer Template",
+        "config_attrs": [],
+        "attrs": [
+          {
+            "template_id": "1",
+            "created": "2018-01-25T12:30:42.167126+00:00",
+            "label": "temperature",
+            "value_type": "float",
+            "type": "dynamic",
+            "id": 1
+          }
+        ],
+        "id": 1
+      }
+    }
+
+Note that the template ID is 1 (line 27).
+
+To create a template based on it, send the following request to dojot:
+
+.. code-block:: bash
+    :linenos:
+
+    curl -X POST http://localhost:8000/device \
+    -H "Authorization: Bearer ${JWT}" \
+    -H 'Content-Type:application/json' \
+    -d ' {
+      "templates": [
+        "1"
+      ],
+      "label": "device"
+    }'
+
+
+The template ID list on line 6 contains the only template ID configured so far.
+To check out the configured device, just send a GET request to /device:
+
+.. code-block:: bash
+
+    curl -X GET http://localhost:8000/device -H "Authorization: Bearer ${JWT}"
+
+
+Which should give back:
 
-Remember that the token must be set in the request header as a whole, not parts of it. 
-In the example only the first characters are shown for the sake of simplicity.
+.. code-block:: bash
+
+    {
+      "pagination": {
+        "has_next": false,
+        "next_page": null,
+        "total": 1,
+        "page": 1
+      },
+      "devices": [
+        {
+          "templates": [
+            1
+          ],
+          "created": "2018-01-25T12:36:29.353958+00:00",
+          "attrs": {
+            "1": [
+              {
+                "template_id": "1",
+                "created": "2018-01-25T12:30:42.167126+00:00",
+                "label": "temperature",
+                "value_type": "float",
+                "type": "dynamic",
+                "id": 1
+              }
+            ]
+          },
+          "id": "0998",
+          "label": "device_0"
+        }
+      ]
+    }
+
+
+Sending messages
+++++++++++++++++
+
+So far we got an access token and created a template and a device based on it.
+In an actual deploy, the physical device would send messages to dojot with all
+its attributes and their current values. For this tutorial we will send MQTT
+messages by hand to the platform, emulating such physical device. For that, we
+will use mosquitto_pub from Mosquitto project.
+
+.. ATTENTION::
+    Some Linux distributions, Ubuntu in particular, have two packages for
+    `mosquitto`_ - one containing tools to access it (such mosquitto_pub and
+    mosquitto_sub for publishing messages and subscribing to topics) and
+    another one containing the MQTT broker. In this tutorial, only the tools
+    are going to be used. There are some reports of problems that were caused
+    because the broker was mistakenly installed and automatically started,
+    which prevents the MQTT broker instance from dojot's docker-compose to be
```

Comment:
Here we could be, perhaps, a bit more specific?

> .. which prevents docker-compose from configuring external MQTT port forwarding to dojot's own MQTT broker

Reply:
In this section I added a quick instruction asking the user to be sure that there is no extra MQTT brokers running before starting dojot. I think this might be enough.